### ISSUE TITLE
Introduce a button to force-trigger the minification process

### ISSUE THREAD

**Issue by shadyvb** [xwp/wp-dependency-minification#45]
Relates to #43

**Comment by lkraav** [comment]
When I'm setting up fresh new sites, I'm noticing I'm not able to get the minification process triggered in a reasonable amount of time by just browsing. Is this force button the thing I'm looking for?

**Comment by westonruter** [comment]
Yes, and/or instead of scheduling an “immediate” minification action, it could do the minification inline (on the fly) if an admin user is browsing the site. This would result in slower initial page loading for admin users, but at least the minification results would appear faster. However, there are unique scripts that only get enqueued when a user is logged in, so actually the minified scripts may not even be the same ones that end users would normally see.

There could be a query_var for force-triggering such inline minification even for non-authenticated users. WordPress could then cURL the siteurl with this query_var to trigger inline minification.

**Comment by lkraav** [comment]
Does anyone here have any quick hack level code stashed right now to achieve any one of these quick minification trigger strategies? Can be totally non-polished.

**Comment by westonruter** [comment]
I do not. My time is limited to look at this plugin.
